### PR TITLE
Fix typos and wording in basics.md

### PR DESCRIPTION
**Pull request by FRickReich**
Fixed some typos, rewrote some of the sentences.

### CODE REVIEW

File: docs/basics.md [hunk 1]
```diff
@@ -16,21 +16,21 @@ Scenario('check Welcome page on site', (I) => {
 
 Tests are expected to be written in **ECMAScript 7**.
 
-Each test is described inside a `Scenario` function with `I` object passed into it.
-I object is an **actor**, an abstraction for a testing user. I is a proxy object for currently enabled **Helpers**.
+Each test is described inside a `Scenario` function with the `I` object passed into it.
+I object is an **actor**, an abstraction for a testing user. The I is a proxy object for currently enabled **Helpers**.
```
Comment (pablopaul) [inline docs/basics.md]: @FRickReich also in the beginning, "The I object…"

File: docs/basics.md [hunk 2]
```diff
@@ -16,21 +16,21 @@ Scenario('check Welcome page on site', (I) => {
...
 
 ## Architecture
 
-CodeceptJS bypasses execution commands to helpers. Depending on helper enabled your tests will be executed differently. If you need cross-browser support you should choose Selenium-based WebDriver or Protractor, if you are interested in speed - use Chrome-based Puppeteer, or Electron-based Nightmare. Those engines can run tests in window mode or headlessly and doesn't require additional tools to be installed.
+CodeceptJS bypasses execution commands to helpers. Depending on the helpers enabled state, your tests will be executed differently. If you need cross-browser support you should choose Selenium-based WebDriver or Protractor, if you are interested in speed - you should use Chrome-based Puppeteer, or Electron-based Nightmare. Those engines can run tests in window mode or headlessly and doesn't require additional tools to be installed.
```
[the remark this answers is not in the document]
Comment (pablopaul) [reply docs/basics.md]: @FRickReich TestCafé can be also mentioned for people interested in cross-browser support and I would remove the Nightmare recommendation as its "oudated".

File: docs/basics.md [hunk 3]
```diff
@@ -16,21 +16,21 @@ Scenario('check Welcome page on site', (I) => {
...
 
 ![architecture](https://codecept.io/img/architecture.svg)
 
-All helpers share the same API so it's easy to migrate tests from one backend to other.
-However, because of difference in backends and their limitations, they are not guaranteed to be compatible with each other. For instance, you can't set request headers in WebDriver or Protractor, but you can do so in Puppteer or Nightmare.
+All helpers share the same API, so it's easy to migrate tests from one backend to another.
+However, because of the difference in backends and their limitations, they are not guaranteed to be compatible with each other. For instance, you can't set request headers in WebDriver or Protractor, but you can do so in Puppteer or Nightmare.
 
-**Pick one helper, as it defines how tests are executed.** If requirements change it's easy to migrate to another, but don't use few helpers of same kind at once.
+**Pick one helper, as it defines how tests are executed.** If requirements change it's easy to migrate to another, but don't use too many helpers of same kind at once.
```
Comment (pablopaul) [inline docs/basics.md]: @DavertMik whats the purpose of this sentence: " but don't use too many helpers of same kind at once."? Does not make sense to me.

Comment (FRickReich) [reply docs/basics.md]: I'll look into it again, I just noticed there are a lot of "The"'s missing, and some other strange wordings, so I wanted to clean that up a bit.

Comment (pablopaul) [reply docs/basics.md]: @FRickReich perfect, thanks alot

**Comment by PeterNgTr** [comment]
@FRickReich could you resolve the conflict?

**Comment by DavertMik** [comment]
Closed in favor of  #2030

**Pull request closed without merging**